Kubernetes/windows/debug/collectlogs.ps1

```diff
@@ -65,5 +65,21 @@ if ($res)
     docker ps -a > docker.txt
 }
 
+netsh int ipv4 sh excludedportrange TCP > reservedports.txt
+netsh int ipv4 sh excludedportrange UDP >> reservedports.txt
+
```

Comment:
Is ICMP also valid?

Reply:
No, it isn't; I have just checked. I don't think we can reserve any ports for ICMP.